v3/tutorials/01-create-your-first-substrate-chain/index.mdx

```diff
@@ -185,7 +185,7 @@ To compile the Substrate Node Template:
 1. Clone the node template repository using the version `latest` branch by running the following command:
 
    ```
-   git clone -b latest --depth 1 https://github.com/substrate-developer-hub/substrate-node-template
+   git clone https://github.com/substrate-developer-hub/substrate-node-template
```

Comment:
this is not as we discussed right? we in fact want yo use the `latest` tag explicitly. that we we can iterate on `main` and when ready, shift all contact by just moving the `latest` tag.

Reply:
we also absolutely should use depth = 1. why not?

Reply:
As per my note above:
![Screen Shot 2021-11-26 at 2 28 03 PM](https://user-images.githubusercontent.com/23283108/143588186-9580e43e-9cf9-4bc6-be2e-dc64c6266551.png)

Yes I agree, using `--depth 1` is not a bad idea.

Reply:
This line points to the default branch (`master`) and is incorrect. It should be `latest` in this PR ***OR*** `monthly-2021-11-1`

Reply:
Are you waiting on some template companion to get the tag in place first? If so, please mention it explicitly in the original PR comment :pray:

Reply:
Right now master is using `monthly-2021-11-1` and `latest` is using `monthly-2021-10`. So given this PR is work done in parallel to updating our solutions to `monthly-2021-11-1`, I wanted to make sure the write-up corresponds to the solution code. 

You've mentioned different plans for ensuring our template is always using `latest` which is what we need to follow up with. But for now, at least with this PR, our users won't have issues when going through tutorials.

As for figuring out the way to maintain our template to `latest`, we need to prioritize this as top priority this week so that we can drop the need to keep tagging solutions and write-ups each month.